.github/workflows/test_publish.yml

```diff
@@ -0,0 +1,70 @@
+name: Integration Test when Publishing to External Data Warehouses
+
+on:
+  pull_request:
+    types: 
+    - closed
+    branches:
+    - master
+    paths:
+    - renku/core/commands/providers/*.py
+
+jobs:
+  test-linux-integration:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Since this is mainly checking our API assumptions, do we need this matrix?

Reply:
To be clear, it will be `python-version: [3.6]`.